tests/test_gui.py

```diff
@@ -0,0 +1,904 @@
+        "description": "This asset was created by the Panel UI automation",
+        "sg_status_list": "ip",
+        "sg_asset_type": "Character",
+        "task_template": asset_task_template,
+    }
+    asset = credentials.create("Asset", asset_data)
+
+    # Get the publish_file_type id to be passed in the publish creation
+    published_file_type_filters = [["code", "is", "Image"]]
+    published_file_type = credentials.find_one(
+        "PublishedFileType", published_file_type_filters
+    )
+
+    # File to publish
+    file_to_publish = os.path.join(
+        os.path.expandvars("${TK_TEST_FIXTURES}"), "files", "images", "sven.png"
+    )
+
+    # Create a version an upload to it
+    version_data = {
+        "project": context,
+        "code": "sven.png",
+        "description": "This version was created by the Panel UI automation",
+        "entity": asset,
+    }
+    version = credentials.create("Version", version_data)
+    # Upload a version to the published file
+    credentials.upload("Version", version["id"], file_to_publish, "sg_uploaded_movie")
+    # Create a published file
+    # Find the model task to publish to
+    filters = [
+        ["project", "is", context],
+        ["entity.Asset.code", "is", asset["code"]],
+        ["step.Step.code", "is", "model"],
+    ]
+    fields = ["sg_status_list"]
+    model_task = credentials.find_one("Task", filters, fields)
+    publish_data = {
+        "project": context,
+        "code": "sven.png",
+        "name": "sven.png",
+        "description": "This file was published by the Panel UI automation",
+        "published_file_type": published_file_type,
+        "path": {"local_path": file_to_publish},
+        "entity": asset,
+        "task": model_task,
+        "version_number": 1,
+        "version": version,
+        "image": file_to_publish,
+    }
+    publish_file = credentials.create("PublishedFile", publish_data)
+
+    # Assign a task to the current user
+    # Find current user
+    user = get_toolkit_user()
+    current_user = credentials.find_one(
+        "HumanUser", [["login", "is", str(user)]], ["name"]
+    )
+    # Assign current user to the task model
+    credentials.update(
+        "Task",
+        model_task["id"],
+        {
+            "content": "Model",
+            "task_assignees": [{"type": "HumanUser", "id": current_user["id"]}],
+        },
+    )
+
+    return (model_task, publish_file, version)
+
+
+# This fixture will launch tk-run-app on first usage
+# and will remain valid until the test run ends.
+@pytest.fixture(scope="session")
+def host_application(context, assetTask):
+    """
+    Launch the host application for the Toolkit application.
+
+    TODO: This can probably be refactored, as it is not
+    likely to change between apps, except for the context.
+    One way to pass in a context would be to have the repo being
+    tested to define a fixture named context and this fixture
+    would consume it.
```

Comment:
Just indenting the following TODO lines. In PyCharm it will then recognise all of those lines as a continuation of the TODO. 
```suggestion
    TODO: This can probably be refactored, as it is not
     likely to change between apps, except for the context.
     One way to pass in a context would be to have the repo being
     tested to define a fixture named context and this fixture
     would consume it.
```